packages/lib/package/components/MazSelect.vue

```diff
@@ -130,6 +130,7 @@
       },
     },
     itemHeight: { type: Number, default: 40 },
+    itemWidth: { type: Number, default: 140 },
```

Comment:
```suggestion
    itemWidth: { type: Number, default: undefined },
```

Don't set a default value to not change the initial behavior, by default the width of the item/option should be 100% of the parent (list)